lib/Activity/Setting.php

```diff
@@ -49,35 +35,35 @@ public function getPriority(): int {
 		return 70;
 	}
 
-	/**
-	 * @return bool True when the option can be changed for the stream
-	 * @since 11.0.0
-	 */
 	public function canChangeStream(): bool {
 		return false;
 	}
 
-	/**
-	 * @return bool True when the option can be changed for the stream
-	 * @since 11.0.0
-	 */
 	public function isDefaultEnabledStream(): bool {
 		return true;
 	}
 
-	/**
-	 * @return bool True when the option can be changed for the mail
-	 * @since 11.0.0
-	 */
 	public function canChangeMail(): bool {
 		return false;
 	}
 
-	/**
-	 * @return bool True when the option can be changed for the stream
-	 * @since 11.0.0
-	 */
 	public function isDefaultEnabledMail(): bool {
 		return false;
 	}
+
+	public function canChangeNotification(): bool {
+		return false;
+	}
+
+	public function isDefaultEnabledNotification(): bool {
+		return false;
+	}
+
+	public function getGroupIdentifier(): string {
+		return 'other';
+	}
+
+	public function getGroupName(): string {
+		return 'Other activities';
```

Comment:
Should be translated?

Reply:
Since all `canChange*` are false, the entry will never display.
And at the same time it should be the group of server I think